### ISSUE TITLE
Also modify `meta` in `fractal workflow add-task`

### ISSUE THREAD

**Issue by jluethi** [fractal-analytics-platform/fractal#346]
I'm trying to run the cellpose_segmentation task on the FMI setup which has no gpu (and therefore, we don't have the gpu specified as a possible option of executors).

I used the new ability to edit the executor of a workflow task to cpu-low, which is available. But the workflow still fails early on with this error message:

(redacted error message)
```
ERROR:    Exception in ASGI application
 in recursive_task_assembly
    raise ValueError(msg)
ValueError: this_task.executor='gpu' is not in data_flow_kernel.executors.keys()=dict_keys(['cpu-low', 'cpu-mid', 'cpu-high', '_parsl_internal'])
```

When checking the workflow, the relevant workflow task looks like this:

```
   {
      "workflow_id": 2,
      "task_id": 5,
      "order": 5,
      "meta": null,
      "args": {
        "args": {
          "labeling_channel": "A01_C01",
          "labeling_level": 2,
          "ROI_table_name": "well_ROI_table"
        },
        "meta": {
          "executor": "cpu-low"
        }
      },
      "id": 16,
      "task": {
        "name": "Cellpose Segmentation",
        "command": "/opt/fmi/gliberal/fractal/fractal-demos/examples-v1/server/FRACTAL_ROOT/.fractal/fractal-tasks-core0.4.6/venv/bin/python /opt/fmi/gliberal/fractal/fractal-demos/examples-v1/server/FRAC
TAL_ROOT/.fractal/fractal-tasks-core0.4.6/venv/lib/python3.8/site-packages/fractal_tasks_core/cellpose_segmentation.py",
        "source": "pip:fractal-tasks-core==0.4.6",
        "input_type": "zarr",
        "output_type": "zarr",
        "default_args": {
          "labeling_channel": "A01_C01"
        },
        "meta": {
          "executor": "gpu",
          "parallelization_level": "image"
        },
        "id": 5
      }
    },

```

It appears the `recursive_task_assembly` of parsl uses the default executor, not the one provided to the workflow task. If this is just a parsl issue, then the new slurm executors should solve it. But would be good to have a test in place to check that a task that defaults to have a gpu executor can be used in a setup without gpu executors when the executor is overwritten in the workflow task to something that is available.

**Comment by tcompa** [comment]
Renamed and move to client repo

**Comment by tcompa** [comment]
Any preference about the required interface?
Right now it is
```
fractal workflow add-task ... --args-file /some/args/file.json
fractal workflow edit-task ... --json-file /some/json/file.json
```
with `/some/args/file.json` having no fractal-specific keywords, while `/some/json/file.json` must have `args` and/or `meta` keys.

Let's aim for a uniform interface of the two commands. Options:
1. They both take a `json-file`-like argument.
2. They both take two separate arguments (`args-file` and `meta-file`).

**Comment by jluethi** [comment]
I think we'll mostly just change the args, not the meta. Thus, going with option 2 and having a meta json file we can provide for the few edge-cases where we want to modify it sounds good to me. Also keeps the args & meta separated well.

Then, it could look like this
```
fractal workflow add-task ... --args-file /some/args/file.json --meta-file /some/meta/file.json 
``` 

With both files being optional of course

**Comment by tcompa** [comment]
Released in `1.0.0a2` and tested with fractal-demos (e.g. with https://github.com/fractal-analytics-platform/fractal-demos/commit/4de14f86fafa49a2d43570bafee61aa17132bc18).
